frontend/src/components/professor/ActivityAssessmentList/ActivityAssessmentList.js

```diff
@@ -42,21 +44,58 @@ export default function ActivityAssessmentList() {
       })
   }, [])
 
-  return (
-    <Content>
-      <h1 style={{ marginLeft: '20px', paddingTop: '20px' }}>Aktywności do sprawdzenia</h1>
-      <Col style={{ paddingTop: '50px' }}>
-        {activityList === undefined ? (
+  if (activityList === undefined) {
+    return (
+      <Content>
+        <h1 style={{ marginLeft: '20px', paddingTop: '20px' }}>Aktywności do sprawdzenia</h1>
+        <Col style={{ paddingTop: '50px' }}>
           <Loader />
-        ) : activityList == null ? (
+        </Col>
+      </Content>
+    )
+  }
+  if (activityList === null) {
+    return (
+      <Content>
+        <h1 style={{ marginLeft: '20px', paddingTop: '20px' }}>Aktywności do sprawdzenia</h1>
+        <Col style={{ paddingTop: '50px' }}>
           <p className={'text-center text-danger h4'}>{ERROR_OCCURRED}</p>
-        ) : (
-          activityList.map((activity) => {
+        </Col>
+      </Content>
+    )
+  }
+
+  if (activityList.length > 0 && activityList.filter((activity) => activity.toGrade > 0)) {
+    return (
+      <Content>
+        <h1 style={{ marginLeft: '20px', paddingTop: '20px' }}>Aktywności do sprawdzenia</h1>
+        <Col style={{ paddingTop: '50px' }}>
+          {activityList.map((activity) => {
             const listActivity = activity.activity.activity
             const toGrade = activity.activity.toGrade
-            return <ActivityListItem key={listActivity.activityName} activity={listActivity} toGrade={toGrade} />
-          })
-        )}
+            return (
+              <>
+                <ActivityListItem key={listActivity.activityName} activity={listActivity} toGrade={toGrade} />
+              </>
```

Comment:
Gdy zwracasz jeden element to nie musisz dodawać pustego tagu okalającego. Dodatkowo jest error w konsoli, który dotyczy brakującego key'a i zniknie gdy wyrzucisz ten pusty tag